packages/camera/camera_windows/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: camera_windows
 description: A Flutter plugin for getting information about and controlling the camera on Windows.
 repository: https://github.com/flutter/packages/tree/main/packages/camera/camera_windows
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+camera%22
-version: 0.2.1+9
+version: 0.3.0
```

Comment:
This would indicate a breaking change by Dart/pub convention; a new feature would be `0.2.2`

Reply:
Chnaged to 2.2.0